src/pages/UpdateServers.js

```diff
@@ -386,6 +385,11 @@ class UpdateServers extends BaseUpdateWizardPage {
     // it to complete before moving on.
     await this.props.updateGlobalState('model', model);
 
+    // save the encryptKey in global cache so it could be retrieved later
```

Comment:
I believe that the constructor needs something like:
```
  this.state = {
     ...this.state
```
in order to be consistent with the other components that extend the baseupdatewizard

Reply:
ok, will update.